smurfs/src/index.js

```diff
@@ -1,6 +1,22 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import { createStore, applyMiddleware } from "redux"
+import { Provider } from "react-redux"
+import thunk from "redux-thunk"
+import "bootstrap/dist/css/bootstrap.min.css"
 import "./index.css";
-import App from "./components/App";
 
-ReactDOM.render(<App />, document.getElementById("root"));
+import reducer from './reducers'
+
+// import App from "./components/App";
+import SmurfList from './components/SmurfList'
+import SmurfForm from "./components/SmurfForm";
+
+const store = createStore(reducer, applyMiddleware(thunk))
+
+ReactDOM.render(
+    <Provider store={store}>
+        <SmurfForm />
+        <SmurfList />
```

Comment:
Tossing `App.js` out the window! You can go ahead and delete the file if you don't need it anymore '' '